### ISSUE TITLE
NVDA Doesn't Register Fast Focus Changes On Dropdown

### ISSUE THREAD

**Issue by srm985** [nvaccess/nvda#8749]
### Steps to reproduce:

1. Open HTML dropdown.
2. Select value.
3. Remove focus from dropdown very rapidly via blur or switch focus to another element.

[CodePen Example](https://codepen.io/srm985/pen/Kxeeeg)

### Actual behavior:

NVDA does not read the selected dropdown value unless a delay is given before removing focus.

### Expected behavior:

NVDA should read the selected element no matter how fast focus is removed.

#### NVDA Installed/portable/running from source:

Installed

#### NVDA version:

2018.2.1

#### Windows version:

Windows 10

#### Does the issue still occur after restarting your PC?

Yes

#### Have you tried any other versions of NVDA?

2013, 2016

#### Browsers Tested

* Chrome 69
* IE11

**Comment by DrSooom** [comment]
@srm985: Please list all your web browsers (including version) where this issue occurred. Thanks.

**Comment by srm985** [comment]
Tested in Chrome 69 and IE11.

**Comment by Adriani90** [comment]
I cannot reproduce this behavior with NVDA 2019.3 Beta 1, on Chrome 79.
However, when focusing a combo box and moving with arrow key through the options without expanding the combo box, NVDA seems to loose focus and reports the window title. I have to press alt+tab twice or so to regain focus on the combo box.
cc: @feerrenrut

**Comment by Adriani90** [comment]
It seems it does have something to do with the timeout that is set for the focus to move, in the combo box with 500 ms timeout, NVDA does not announce the focus movements if you move very fast. In the combo box without timeout, NVDA announces the window title instead of the element selected when arrowing through the combo box in focus mode.
Maybe @jcsteh has some suggestions here.

**Comment by jcsteh** [comment]
Honestly, I don't think this is reasonable. For a screen reader user, focus is paramount. Screen readers deliberately only report information about the focused control. If the focus moves before some change in the previously focused control is reported, the focus will take precedence. Doing otherwise almost always results in redundant information being reported or sluggish performance.

Aside from anything else, moving focus as soon as a selection has been made is problematic for keyboard users because they might need to move through multiple options to get to the one they want. If you move focus, they won't be able to get to any option other than the one immediately below or above the current selection without a lot of tedium and frustration.

IMO, this should be a wontfix.

**Comment by jcsteh** [comment]
Closing as wontfix per the above comment. I doubt the core devs will disagree, but if they do, my apologies; please reopen.
